app/src/main/java/org/fossasia/pslab/communication/ScienceLab.java

```diff
@@ -223,6 +226,116 @@ public void close() {
         }
     }
 
+    public Map<String, double[]> captureOne(String channel, int samples, double timeGap) {
+        return this.captureFullSpeed(channel, samples, timeGap, null, null);
+    }
+
+    public Map<String, double[]> captureTwo(int samples, double timeGap, String traceOneRemap) {
+        if (traceOneRemap == null) traceOneRemap = "CH1";
+        this.captureTraces(2, samples, timeGap, traceOneRemap, null, null);
+        SystemClock.sleep((long) (1e-6 * this.samples * this.timebase + 0.1));
```

Comment:
The constant as per ScienceLab.py is 0.01 and SystemClock.sleep takes time in milliseconds whereas time.sleep in python in seconds. Multiply 1000.